docs-v2/_docs/running-standalone.md

```diff
@@ -298,6 +298,29 @@ matching the URL exists. For example if a file exists
 `/things/myfile.html` then hitting
 `http://<host>:<port>/things/myfile.html` will serve the file.
 
+## Packaging the stubs into a standalone JAR
+
+If you want to package your stubs into the standalone JAR, so you can distribute an executable JAR with all the stubs intact, you can do this using the `--load-resources-from-classpath` option.
+
+For example, let's say have the following directory structure:
+
+```
+src/main/resources
+src/main/resources/wiremock-stuff
+src/main/resources/wiremock-stuff/__files
+src/main/resources/wiremock-stuff/mappings
+```
+
+You could then run the packaged JAR as:
+
+```
+java -jar custom-wiremock.jar --load-resources-from-classpath --root-dir 'wiremock-stuff'
```

Comment:
I feel like these two parameters are (or ought to be) mutually exclusive - either I'm loading from the classpath or file system. What happens if I specify both?

Reply:
Ah so root dir is used to inform the directory on the classpath at that point

Reply:
OK, I see.

As a way of avoiding some confusion, perhaps it'd be worth considering `load-resources-from-classpath` taking the root path as its parameter e.g. `--load-resources-from-classpath=wiremock-stuff`?

Reply:
Good thinking - amended :+1: